packages/app-mobile/components/ButtonWithTooltip.tsx

```diff
@@ -0,0 +1,197 @@
+//
+// A button with a long-press action. Long-pressing the button displays a tooltip
+//
+
+const React = require('react');
+import { ReactNode } from 'react';
+import { themeStyle } from '@joplin/lib/theme';
+import { Theme } from '@joplin/lib/themes/type';
+import { useState, useMemo, useCallback, useRef } from 'react';
+import {
+	View, Text, Pressable, ViewStyle, PressableStateCallbackType,
+	StyleProp, StyleSheet, LayoutChangeEvent, LayoutRectangle, Animated, AccessibilityState, AccessibilityRole,
+} from 'react-native';
+import { Menu, MenuOptions, MenuTrigger, renderers } from 'react-native-popup-menu';
+
+type ButtonClickListener = ()=> void;
+interface ButtonProps {
+	onClick: ButtonClickListener;
+
+	// Accessibility label and text shown in a tooltip
+	description: string;
+
+	children: ReactNode;
+
+	// [theme] can either be a themeId or a [Theme] object.
+	theme: Theme|number;
```

Comment:
Isn't it alway an ID? Also that should generally be called themeId to make it clear what it is.